src/coretelephony.cs

```diff
@@ -228,8 +228,6 @@ interface CTSubscriberDelegate {
 	partial interface CTSubscriber {
 		[Export ("carrierToken")]
 		[NullAllowed]
-		[Deprecated (PlatformName.iOS, 11, 0)]
-		[Deprecated (PlatformName.MacCatalyst, 13, 1)]
```

Comment:
They removed the deprecation?

Reply:
It's not deprecated right now at least: https://developer.apple.com/documentation/coretelephony/ctsubscriber/1620318-carriertoken?language=objc

Also note that xtro didn't complain.